backend/src/main/java/com/bakdata/conquery/models/jobs/ImportJob.java

```diff
@@ -199,68 +145,76 @@ private Import createImport(PreprocessedHeader header, Map<String, ColumnStore>
 
 		imp.setColumns(importColumns);
 
-		namespace.getWorkerHandler().sendToAll(new AddImport(imp));
 		return imp;
 	}
 
-	/**
-	 * Group entities by their global bucket id.
-	 */
-	private Map<Integer, List<String>> groupEntitiesByBucket(Set<String> entities, int bucketSize) {
-		return entities.stream()
-					   .collect(Collectors.groupingBy(entity -> namespace.getBucket(entity, bucketSize)));
+	@Override
+	public void execute() throws JSONException, InterruptedException, IOException {
 
-	}
 
-	/**
-	 * select, then send buckets.
-	 */
-	private Map<WorkerId, Set<BucketId>> sendBuckets(Map<String, Integer> starts, Map<String, Integer> lengths, Import imp, Map<Integer, List<String>> buckets2LocalEntities, ColumnStore[] storesSorted) {
+		log.trace("Updating primary dictionary");
 
-		final Map<WorkerId, Set<BucketId>> newWorkerAssignments = new HashMap<>();
+		assignResponsibleWorker();
```

Comment:
muss das überhaupt aufgerufen werden?

Reply:
Ja, wir verteilen die Buckets immer noch nach round-robin. Die Buckets über hashing zu assignen habe ich für diesen PR ausgegeben, weil das komplexität in den Tests nach sich zieht die ich gerade noch nicht lösen will.